met-web/tests/unit/components/engagement/form/edit/EngagementForm.Edit.Two.test.tsx

```diff
@@ -145,7 +145,7 @@ describe('Engagement form page tests', () => {
 
         await waitFor(() => {
             expect(screen.getByDisplayValue('Test Engagement')).toBeInTheDocument();
-            expect(container.querySelector('span.MuiSkeleton-root')).toBeNull();
+            expect(container.querySelector('span.MuiSkeleton-root'));
```

Comment:
Just curious: what does calling `expect` without a matcher function checks for?

Reply:
Its just checking if the element exists.